### DISCUSSION TITLE
openPilotLog 0.1

### DISCUSSION THREAD

**Discussion by fiffty-50** [fiffty-50/openPilotLog#98] [General]
### v0.1
This is a first beta release aimed at testing and finding bugs. This is still an early development build, so it is not recommended to use it for storing your actual logbook data. It is intended to be used for testing and experimenting. Please not that there are no guarantees for database compatibility between this build and version 1.0 since significant design changes may still be made.

### Working:
- Flight Logging
- Simulator Logging
- Aircraft Logging
- Pilot Logging
- Export to CSV

### Not working / incomplete:
- Export to PDF
- Import from CSV
- Logging using local time (UTC only supported)
- Aircraft template database incomplete

<hr /><em>This discussion was created from the release <a href='https://github.com/fiffty-50/openPilotLog/releases/tag/'>openPilotLog 0.1</a>.</em>

**Comment by mumbricht** [comment]
What is the state of this project?

**Reply by fiffty-50** [reply]
Development is ongoing, currently undergoing a major [database overhaul](https://github.com/fiffty-50/openPilotLog/tree/database-redesign) focussed on creating a more extensible database layout.
